backends-common/elasticsearch/src/test/java/org/apache/james/backends/es/ElasticSearchIndexerTest.java

```diff
@@ -144,15 +143,16 @@ public void deleteByQueryShouldWorkOnSingleMessage() throws Exception {
         testee.index(messageId, content);
         elasticSearch.awaitForElasticSearch();
         
-        testee.deleteAllMatchingQuery(termQuery("property", "1")).get();
+        testee.deleteAllMatchingQuery(termQuery("property", "1"));
         elasticSearch.awaitForElasticSearch();
         
-        try (Client client = getESClient()) {
-            SearchResponse searchResponse = client.prepareSearch(INDEX_NAME.getValue())
-                    .setTypes(TYPE_NAME.getValue())
-                    .setQuery(QueryBuilders.matchAllQuery())
-                    .get();
-            assertThat(searchResponse.getHits().getTotalHits()).isEqualTo(0);
+        try (RestHighLevelClient client = getESClient()) {
+            await().atMost(Duration.TEN_SECONDS)
```

Comment:
use any CALMLY_AWAIT please

Reply:
I can't find any CALMLY_AWAIT that I can import (I don't want to import non related dependencies), so I created a new one

Reply:
check it out 94f9f7a